<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Sakkal Majalla&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="Home">
       <widget class="QRadioButton" name="WafRadio">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>140</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>WAF Config</string>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="WAFbtn">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>360</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="FrameRadio">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>190</y>
          <width>121</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Framework</string>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="helpbtn">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>400</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>help</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>401</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="assets/features.qrc">:/~/pentesting.png</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>-10</x>
          <y>150</y>
          <width>261</width>
          <height>301</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="assets/features.qrc">:/~/hacker.png</pixmap>
        </property>
       </widget>
       <widget class="QWidget" name="chooseProject" native="true">
        <property name="geometry">
         <rect>
          <x>244</x>
          <y>240</y>
          <width>281</width>
          <height>56</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLineEdit" name="projectName">
           <property name="placeholderText">
            <string>Project Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="filebtn">
           <property name="text">
            <string>...</string>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="WAF">
       <widget class="QPushButton" name="BackHomebtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>400</y>
          <width>161</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Savebtn">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>400</y>
          <width>141</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="FrameWork">
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <widget class="QPushButton" name="BackHomebtn_2">
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QTabWidget" name="Tools">
          <property name="tabPosition">
           <enum>QTabWidget::North</enum>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::Rounded</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="documentMode">
           <bool>true</bool>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
          <property name="tabBarAutoHide">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="recogn_tab">
           <attribute name="title">
            <string>Recon</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="1" column="0">
             <widget class="QRadioButton" name="singleRadio">
              <property name="text">
               <string>Single Domain</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="2">
             <widget class="QRadioButton" name="listRadio">
              <property name="text">
               <string>List of Domain</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="3">
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Options</string>
              </property>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QCheckBox" name="collect_subdomain">
                 <property name="text">
                  <string>Collect subdomain</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QCheckBox" name="Live_subdomain">
                 <property name="text">
                  <string>Live subdomain</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QCheckBox" name="Screenshot">
                 <property name="text">
                  <string>Screenshot</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QCheckBox" name="Endpoint">
                 <property name="text">
                  <string>End points</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QCheckBox" name="Js_files">
                 <property name="text">
                  <string>JS Files</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QCheckBox" name="Parameter">
                 <property name="text">
                  <string>Parameter</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QWidget" name="widget_single_domain" native="true">
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <widget class="QLineEdit" name="target_line">
                 <property name="placeholderText">
                  <string>Enter Target</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QWidget" name="widget_list_domain" native="true">
                 <layout class="QFormLayout" name="formLayout_2">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Choose File</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QToolButton" name="toolButton">
                    <property name="text">
                     <string>...</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="reconbtn">
              <property name="text">
               <string>Submit</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="attack_tab">
           <attribute name="title">
            <string>Attack</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="assets/features.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>FrameRadio</sender>
   <signal>clicked()</signal>
   <receiver>chooseProject</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>WafRadio</sender>
   <signal>clicked()</signal>
   <receiver>chooseProject</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>singleRadio</sender>
   <signal>clicked()</signal>
   <receiver>target_line</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listRadio</sender>
   <signal>clicked()</signal>
   <receiver>widget_list_domain</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listRadio</sender>
   <signal>clicked()</signal>
   <receiver>widget_single_domain</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>singleRadio</sender>
   <signal>clicked()</signal>
   <receiver>widget_single_domain</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>28</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>singleRadio</sender>
   <signal>clicked()</signal>
   <receiver>widget_list_domain</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listRadio</sender>
   <signal>clicked()</signal>
   <receiver>target_line</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
